apps/logging/loggers.py

```diff
@@ -15,34 +26,94 @@
 
 
 def log_global_state_metrics(group_timestamp=None):
-    '''
+    """
     For use in apps/logging/management/commands/log_global_metrics.py management command
     NOTE:  print statements are for output when run via Jenkins
-    '''
+    """
     print("---")
     print("---RUNNING DJANGO COMMAND:  log_global_state_metrics")
     print("---")
 
+    start_time = datetime.utcnow().timestamp()
+
     crosswalk_counts = check_crosswalks()
+
+    access_token_counts = get_access_token_counts()
+
+    grant_counts = get_grant_counts()
+
     application_counts = get_application_counts()
-    require_demographic_scopes_count = get_application_require_demographic_scopes_count()
 
-    log_dict = {"type": "global_state_metrics",
-                "group_timestamp": group_timestamp,
-                "real_bene_cnt": crosswalk_counts.get('real', None),
-                "synth_bene_cnt": crosswalk_counts.get('synthetic', None),
-                "global_apps_active_cnt": application_counts.get('active_cnt', None),
-                "global_apps_inactive_cnt": application_counts.get('inactive_cnt', None),
-                "global_apps_require_demographic_scopes_cnt": require_demographic_scopes_count, }
+    require_demographic_scopes_count = (
+        get_application_require_demographic_scopes_count()
+    )
+
+    elapsed_time = round(datetime.utcnow().timestamp() - start_time, 3)
+
+    log_dict = {
+        "type": "global_state_metrics",
+        "group_timestamp": group_timestamp,
+        "real_bene_cnt": crosswalk_counts.get("real", None),
+        "synth_bene_cnt": crosswalk_counts.get("synthetic", None),
```

Comment:
Is the synthetic bene count... the number of synthetic bene's currently being used?

Reply:
@nbragdon Those two counts are from the Crosswalk table.  This is our largest type of count. It would include all benes that had a successful medicare-gov login. For those that have successfully authenticated, we need the Grant related counts (a subset of the crosswalk counts).

Reply:
I was just curious why we wanted to calculate the synthetic bene count... since we already know the total number of synthetic bene's (40k). What fine grained information is this giving us?

Reply:
> I was just curious why we wanted to calculate the synthetic bene count... since we already know the total number of synthetic bene's (40k). What fine grained information is this giving us?

@nbragdon The 40K is the total synth benes that are available to BB2 via SLSx/Medicare.gov.  This counts the number of them that have connected or being used in BB2.

For example, when running this last time in Sandbox, the `synth_bene_cnt` (in the Crosswalk table) was 14,633.